core/com.b2international.snowowl.datastore/src/com/b2international/snowowl/datastore/cdo/AbstractCDOConflictProcessor.java

```diff
@@ -135,34 +138,33 @@ public Object detachedInTarget(CDOID id) {
 	}
 	
 	@Override
-	public void preProcess(Map<CDOID, Object> sourceMap, Map<CDOID, Object> targetMap, boolean isRebase) {
+	public void preProcess(Map<CDOID, Object> sourceMap, Map<CDOID, Object> targetMap, CDOBranch sourceBranch, CDOBranch targetBranch, boolean isRebase) {
 	}
 	
 	@Override
 	public void postProcess(final CDOTransaction transaction) throws ConflictException {
 		
-		for (final CDOID idToUnlink : idsToUnlink) {
-			final CDOObject objectIfExists = CDOUtils.getObjectIfExists(transaction, idToUnlink);
-			if (objectIfExists != null) {
-				unlinkObject(objectIfExists);
-			}
-		}
+		LOGGER.info("Post-processing merge/rebase operation...");
+		Stopwatch stopwatch = Stopwatch.createStarted();
 		
-		List<MergeConflict> conflicts = FluentIterable.from(conflictRuleProvider.getRules())
+		List<MergeConflict> conflicts = FluentIterable.from(getConflictRules())
 				.transformAndConcat(new Function<IMergeConflictRule, Collection<MergeConflict>>() {
 					@Override
 					public Collection<MergeConflict> apply(IMergeConflictRule input) {
 						return input.validate(transaction);
 					}
 				}).toList();
 
+		LOGGER.info("Post-processing took {}", TimeUtil.toString(stopwatch));
```

Comment:
`Stopwatch.toString()` creates a pretty nice human-readable string. How about using that instead of our custom `TimeUtil.toString()` function.